shortfin/python/lib_ext.h

```diff
@@ -65,6 +65,7 @@ template <typename CppType, typename KeepAlivePatient, typename... Args>
 inline py::object custom_new_keep_alive(py::handle py_type,
                                         KeepAlivePatient &keep_alive,
                                         Args &&...args) {
+  py::set_leak_warnings(false);
```

Comment:
Please move this to the module registration function in lib_ext.cpp. it is a global. 

Agreed this thing is flaky. ASAN does a more reliable job.